apps/mobile-wallet/ios/Alephium/Info.plist

```diff
@@ -91,4 +91,4 @@
     <key>UIViewControllerBasedStatusBarAppearance</key>
     <false/>
   </dict>
-</plist>
+</plist>
```

Comment:
That's weird, there are multiple new lines EOF across the PR, but I guess we don't care :)